src/components/YourCompany/YourCompany.vue

```diff
@@ -62,8 +126,36 @@
             />
           </v-flex>
         </template>
+
+        <template v-if="isAlteration() && hasNewNr">
+          <v-flex xs3 class="mt-6">
+            <v-layout column>
+              <label><strong>Name Request Applicant</strong></label>
+            </v-layout>
+          </v-flex>
+          <v-flex xs6 class="mt-6">
+            <template>
+              <div class="name-request-applicant-info">
+                <span class="font-weight-bold">Name: </span>
+                <span>{{nrApplicant.fullName}}</span>
```

Comment:
This could still fail if nrApplicant is null (per getter). 

One option is to do a v-if="nrApplicant" over this block.

Another option is to change the getter to:
```
return this.getNameRequest?.applicant || {}
```

Reply:
Oh I see, you've initialized applicant.

As long as state.applicant that doesn't get overwritten with null, that should be OK.

Reply:
So many ways to skin a cat...

Reply:
Your telling me lol
It's good though!